src/helpers/helpers.js

```diff
@@ -5,9 +5,9 @@ import jwtDecode from 'jwt-decode';
 class Helpers {
   static setToken(token) {
     if (localStorage.token) {
-      localStorage.removeItem('token');
+      localStorage.removeItem("token");
```

Comment:
Single quote over double. Please configure your IDE to stop formatting when you save a file or to stop changing single quotes to double